change/@itwin-tree-widget-react-606e3925-b701-4946-8d58-84282dd89762.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "resolve cves",
```

These messages appear in consumer-facing changelog files - they should have a meaning to consumers and be properly formatted.

```suggestion
  "comment": "Bump dependencies.",
```